packages/fields/package.json

```diff
@@ -62,7 +62,7 @@
     "prop-types": "^15.7.2",
     "query-string": "^6.8.1",
     "react": "^16.13.1",
-    "react-color": "^2.17.3",
+    "react-color": "^2.18.1",
```

Comment:
Can this dependency just be removed?

Reply:
You mean the update removed? I can... it's a pretty safe bump and I'd like to update it at some point.

Reply:
I think @timleslie means that given that the Color field is not in `@keystonejs/fields` now, `@keystonejs/fields` shouldn't need to depend on `react-color`